common/djangoapps/track/backends/tests/test_logger.py

```diff
@@ -47,6 +47,22 @@ def test_logger_backend(self):
         self.assertEqual(saved_events[0], unpacked_event)
         self.assertEqual(saved_events[1], unpacked_event)
 
+    def test_logger_backend_unicode_character(self):
+        """
+        When event information contain unicode characters
+        """
+        self.handler.reset()
+
+        event = {'string': '\xd3 \xe9 \xf1'}
```

Comment:
This string does not have a `u` prefix, but the string a few lines down in this same test does. Is that intentional? Please add some comments explaining what this test is supposed to actually test, and how it works.